### DISCUSSION TITLE
Help: Looking to add a floating button to scroll back to the top

### DISCUSSION THREAD

**Discussion by Chriis716** [EvotecIT/PSWriteHTML#239] [Q&A] [answered]
Hi, I'm new to this so please bear with me....I'm looking to add a floating button for the ability to scroll back up on a large page. It it possible to add my own jquery to the script? If so, how would one be able to do this? I'm having a hard time searching through the examples to find something similar that would work. Something as simple as this page https://www.w3schools.com/howto/howto_js_scroll_to_top.asp

**Comment by PrzemyslawKlys** [comment, marked as answer]
While it's possible to add custom js and add your own code you can achieve the same thing with 0 js using just anchors. 

https://github.com/EvotecIT/PSWriteHTML/blob/master/Examples/Example27-SectionsPanelsContainers/Example27-05.ps1#L74

Would that work? Otherwise I can probably provide you a way to do custom code.

I plan to implement this a bit later on: https://codepen.io/bramus/pen/ExaEqMJ

**Reply by Chriis716** [reply]
I was hoping to add a floating button to my file. How would I go about adding custom code to my own script?
